src/sentry/integrations/jira/integration.py

```diff
@@ -378,12 +381,12 @@ def get_issue_type_meta(self, issue_type, meta):
     def get_create_issue_config(self, group, **kwargs):
         fields = super(JiraIntegration, self).get_create_issue_config(group, **kwargs)
         params = kwargs.get('params', {})
-
-        # TODO(jess): update if we allow saving a default project key
-
+        default_project = params.get('project') or self.get_project_defaults(
+            group.project_id).get('project')
```

Comment:
@macqueen this should handle jira's dynamic fields.

Reply:
It might be a little clearer to use the default of the `params.get`. So something like

```py
default_project = params.get(
  'project',
  self.get_project_defaults(group.project_id).get('project')
)
```